You can save ores!

AoKMiKeY opened the pull request:
Added support for saving ores and tided some things up. New version 1.0.1.75

mcd1992 commented:
Looks good, the only changes you need is to make is keep the keybindings default to NONE. This way when someone puts it in a modpack it doesn't conflict with other keybindings and the user gets to pick what keys to bind to.

Also you can put yourself in the mcmod.info contributors if you'd like.

Also thanks for working on this project!

AoKMiKeY commented:
Right, I have fixed that up :) 

Also no problem. Love the mod and I do not mind maintaining it :P I use it myself all the time so I have just gone though and fixed things up. I want to learn how to render items so I can have the ore texture on the button. 

I am going to be added a remove from cofig button soon but I could not be asked to work out how I am going to do it and make it look good.
